src/darwin/Framework/CHIP/MTROTAProviderDelegateBridge.mm

```diff
@@ -574,11 +614,19 @@ bool GetPeerNodeInfo(CommandHandler * commandHandler, const ConcreteCommandPath
                           CHIP_ERROR err = gOtaSender.PrepareForTransfer(fabricIndex, nodeId);
                           if (CHIP_NO_ERROR != err) {
                               LogErrorOnFailure(err);
+                              if (err == CHIP_ERROR_BUSY) {
+                                  Commands::QueryImageResponse::Type busyResponse;
+                                  busyResponse.status = static_cast<OTAQueryStatus>(MTROTASoftwareUpdateProviderOTAQueryStatusBusy);
+                                  busyResponse.delayedActionTime = response.delayedActionTime;
```

Comment:
Oh, and this is still not guaranteeing a non-null delayedActionTime.  We need to be guaranteeing that.

Reply:
if theres no delayedactiontime i will default to 120 secs (spec mandated)

Reply:
done